package.json

```diff
@@ -7,13 +7,12 @@
     "test": "tests"
   },
   "engines": {
-    "node": "^4.2.0"
+    "node": "^6.10.0"
```

Comment:
Does yarn require 6.10? Don't assume its easy for devs to update minor releases (e.g. when using a SaaS provider which might lag a bit). Can we just say 6.x? Yarn itself claims to work with >4.0.0 (https://github.com/yarnpkg/yarn/blob/0.22-stable/package.json#L79)